apps/condo/domains/document/schema/Document.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Generated by `createschema document.Document 'organization:Relationship:Organization:CASCADE; property?:Relationship:Property:CASCADE; category:Relationship:DocumentCategory:CASCADE; file:File;'`
+ */
+const get = require('lodash/get')
+
+const { GQLError } = require('@open-condo/keystone/errors')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema, getById } = require('@open-condo/keystone/schema')
+
+const FileAdapter = require('@condo/domains/common/utils/fileAdapter')
+const { getFileMetaAfterChange } = require('@condo/domains/common/utils/fileAdapter')
+const access = require('@condo/domains/document/access/Document')
+const { ERRORS } = require('@condo/domains/document/constants')
+
+
+const DOCUMENT_FOLDER_NAME = 'document'
+const Adapter = new FileAdapter(DOCUMENT_FOLDER_NAME)
+const fileMetaAfterChange = getFileMetaAfterChange(Adapter)
+
+const Document = new GQLListSchema('Document', {
+    schemaDoc: 'Document with file attached to organization or property. It could be some kind of property documentation, inspection reports or other documents',
+    fields: {
+        organization: {
+            schemaDoc: 'Organization to which the document is attached',
+            type: 'Relationship',
+            ref: 'Organization',
+            isRequired: true,
+            knexOptions: { isNotNullable: true },
+            kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
+        },
+        property: {
+            schemaDoc: 'Property to which the document is attached',
+            type: 'Relationship',
+            ref: 'Property',
+            kmigratorOptions: { null: true, on_delete: 'models.CASCADE' },
+        },
+        category: {
+            schemaDoc: 'Document type',
+            type: 'Relationship',
+            ref: 'DocumentCategory',
+            isRequired: true,
+            knexOptions: { isNotNullable: true },
+            kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
```

Comment:
the same goes for categories. I don't think deleting categories should result in deleting documents